### DISCUSSION TITLE
Streaming Start ignoring framerate-, lens- and HyperSmooth settings

### DISCUSSION THREAD

**Discussion by VProdAustria** [gopro/labs#411] [General]
I am not sure, if its an issue in the firmware control or not - But I am sadly not able to get the GoPro Hero 11 with the latest Labs-Firmware to start the streaming with certain framerate-, lens- and hypersmooth settings. Its always starting with the standard 1080P 50 fps with HyperSmooth = on (Mode 1).

The standard code for the 1080P streaming start without recording is: "oW1mVr1080!W!GL"
Which should mean: Wireless on (oW1), mV (VideoMode), Resolution 1080 (r1080)
"!W" I sadly do not know what it does (I guess its the streaming-trigger) and "!GL" defines the streaming resolution (?)
At least thats the code, the streaming page shows for those settings (1080 / no-rec): https://gopro.github.io/labs/control/rtmp/

**I tried following to add:**
e0 = HyperSmooth off
p25 = 25 FPS
fL0 = Linear Mode

**Total code:** "oW1mVr1080p25fL0e0!W!GL"

But its ignoring those settings. The GoPro 11 Balck still starts the streaming in 1080P 50 fps mode with HyperSmooth = on.
Any chance that I missed something there? If not - Is it possible, to add those settings to the internal streaming mode?

**Comment by dnewman-gpsw** [comment]
Unfortunately Live streaming is limited to the default wide 16x9 image with base stabilization, as there is no live streaming preset.  All the e0p25fL0 commands are modifying the current preset.  When livestream starts, it goes to it own default, ignoring all the presets. 

!W - join WiFi
!GL - Go live Large (1080).

**Reply by VProdAustria** [reply]
Thx for the super fast reply (and the explanation of the two additional commands). Its really sad to hear though, that there is actually no option to define a seperate streaming preset (or to modify the streaming trigger directly). Are there any plans, that such presets are coming in future firmwares? Would make the camera(s) WAY more useful as dedicated streaming cameras. Would love to use them additionally for remote starting points too (skiing, snowboarding etc.) - Where non skilled people are then able to easily start the streaming over the Qr code.

Btw.: Also an "auto-reconnect" for example in case there are short period connection / WiFi issues would be ingenious to have. It happens quite easily that you need a reconnection / streaming restart due to a bad signal for a few seconds - Or even more common: The router you are booked into is getting a new IP lease in the middle of a streaming. Acutally the streaming just stops if its loosing connection to the server (after a connection is established).

**About the FPS, linear mode and stabilisation:** If its using the "default" settings on the camera to stream. Is it maybe possible to modify those settings? In case those are editable, it should be possible - Or are those settings kind of "read only" in the system? Thx again and have a nice day.

LG

**Reply by exokinetic** [reply]
Are there any plans to address modifying the "default" Livestream settings?

**Comment by schieder-karl** [comment]
GoPro HERO 13:

Please add support for using the active camera preset settings in RTMP live streaming (GoPro Labs and Quik), especially:

* Lens mode (Linear, Wide, SuperView, HyperView)
* HyperSmooth settings
* Resolution and frame rate
* Color profile and HDR settings

Currently, the RTMP live stream always uses an internal fixed “Wide” pipeline and ignores most camera settings. This makes the HERO13 unsuitable for professional mobile live streaming applications.

Unfortunately, because of these limitations, I have to return my newly purchased GoPro HERO13, as it does not meet my requirements for mobile live streaming despite its otherwise excellent image quality.
